File: examples/filestream/filestream.js
const fs = require('fs');

var writeStream = fs.createWriteStream('hello.txt');

writeStream.write("Whatchu lookin' at!");

var readStream = fs.createReadStream('hello.txt');

readStream.on("data", function(data) {
    var chunk = data.toString();
    console.log(chunk);
}); 

File: PyJS/modules/filestream.py
class createReadStream:
    def __init__(self, path:str):
        self.path = path

    def chunk(self):
        return open(self.path, "r").read()
    
class createWriteStream:
    def __init__(self, path:str):
        self.path = path

    def write(self, _data:str):
        with open(self.path, "w+") as f:
            f.write(_data)